### DISCUSSION TITLE
use twenty as a cms?

### DISCUSSION THREAD

**Discussion by wc-matteo** [twentyhq/twenty#6438] [Engineering vision]
from what i saw twenty could make a good cms too... is it something that's been discussed or thought about before?

**Comment by FelixMalfait** [comment]
Hey! Probably 6month / 1 year from now you could make a case for it (we have a nice custom graphql/rest api for each object/field, we'll have workflows embedded which most CMS don't have etc.). But honestly  I'd always go with a proper headless CMS if it's a traditional use-case. For example Strapi is a great choice and under MIT license
